roles/pxe/templates/dnsmasq.conf.j2

```diff
@@ -0,0 +1,18 @@
+#disable dns server
```

Comment:
??

Reply:
we use BIND as a slave on the routers, so don't use dnsmasq's dns server

Reply:
(will have a role for the bind slave bit shortly :)